lib/config/presets/internal/monorepo.ts

```diff
@@ -87,7 +87,6 @@ const repoGroups = {
     'https://github.com/dotansimha/graphql-codegen',
   ],
   'graphql-mesh': 'https://github.com/Urigo/graphql-mesh',
-  'graphql-toolkit': 'https://github.com/ardatan/graphql-toolkit', // Deprecated, merged into GraphQL Tools.
```

Comment:
@HonkingGoose FYI I think you can "batch" update commit suggestions using GitHub's web UI in case that's easier than doing them one by one

Reply:
> FYI I think you can "batch" update commit suggestions using GitHub's web UI in case that's easier than doing them one by one

Thank you for the tip! I know I can batch commits together, but that loses the context of the change.

I have no clue how to fix up this merge conflict, so I think the only way for _me_ to proceed is to re-do the work in full on the `main` branch?

Do you have any tips on how to fix this branch?

Reply:
I think I've resolved it - please double check